screens/search.js

```diff
@@ -16,16 +17,23 @@ export default class Search extends Component {
 
   }
 
-  submitSearch = () => {
+  submitSearch = async () => {
     const { text, top } = this.state;
-    if (text) {
-      Animated.timing(top, {
-        duration: 300,
-        toValue: 0,
-      }).start(() => Actions.stock({
-        type: 'replace',
-        ticker: text.toUpperCase()
-      }));
+    try {
+      if (text) {
+        let initStockHistory = await getStockHistory(text, 'day');
```

Comment:
we should change the default to not day anymore. don't do it if it's out of scope though